### ISSUE TITLE
API Key error on stripe terminal

### ISSUE THREAD

**Issue by jbcullis** [stripe/stripe-dotnet#1650]
Windows 10
.Net 4.7.2
Stripe 27.0.0

Upgrading stripe terminal out of beta to v1 and thought i'd upgrade stripe.net and api as well.

Calling the following was fine before the upgrade:

```vb
Return New Stripe.Terminal.ConnectionTokenService().Create(Nothing, New 
     Stripe.RequestOptions With {
    .ApiKey = GetKey(XXXXXX)
})
```

Now, returns the following error:
```
No API key provided. Set your API key using `StripeConfiguration.ApiKey = "<API-KEY>"`. You can generate API keys from the Stripe Dashboard. See https://stripe.com/docs/api/authentication for details or contact support at https://support.stripe.com/email if you have any questions.
```

**Comment by ob-stripe** [comment]
Hi @jbcullis, thanks for reporting this. Indeed, this looks like a bug we introduced in the new version: you must provide an API key via `StripeConfiguration.ApiKey = "sk_..."` even if you provide a per-request API key via `RequestOptions`.

We'll release a fix soon, but in the meantime you can work around this by setting a non-empty value in `StripeConfiguration.ApiKey`.

**Comment by ob-stripe** [comment]
After giving it some more thought, I think the new behavior might be preferable. `RequestOptions.ApiKey` is really meant to override the service's default API key when using the same service to send more than one request.

Since you're creating a service specifically for this request, you can create a client with the API key and pass it to the service:
```vb
Return New Stripe.Terminal.ConnectionTokenService(
  New Stripe.StripeClient(GetKey(XXXXXX))
).Create(Nothing)
```

Would that work for you?

**Comment by jbcullis** [comment]
@ob-stripe Not sure if I'm missing something, I thought the way I had it was recommended for connect applications.
PS. We will be using the StripeConnectAccountId field in the future.

**Comment by ob-stripe** [comment]
The recommended way for Connect applications is to set the global API key in `StripeConfiguration.ApiKey` to the platform account's API key, and set `RequestOptions. StripeAccount` to the ID of the connected account on behalf of which you're executing the request.

(`RequestOptions.StripeConnectAccountId` is the "old" name for the same property. As of v27, it still works, but it's deprecated and will be removed in a future major version.)

**Comment by ob-stripe** [comment]
Just a heads-up that we just released version 27.4.0 which restores the old behavior of being able to create a service with no global API key and only passing API keys via `RequestOptions`.

**Comment by jbcullis** [comment]
Thanks @ob-stripe
